projects/ecoindex_api/README.md

```diff
@@ -33,56 +33,37 @@ With this docker setup you get 6 services running that are enough to make it all
-| API         | `CORS_ALLOWED_ORIGINS`     | `*`                                | See [MDN web doc](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin)                                                                                                                                                                                                                                                                                                                   |
-| API, Worker | `DAILY_LIMIT_PER_HOST`     | 0                                  | When this variable is set, it won't be possible for a same host to make more request than defined in the same day to avoid overload. If the variable is set, you will get a header `x-remaining-daily-requests: 6` in your response. It is used for the POST methods. If you reach your authorized request quota for the day, the next requests will give you a 429 response. If the variable is set to 0, no limit is set |
-| API, Worker | `DATABASE_URL`             | `sqlite+aiosqlite:///./sql_app.db` | If you run your mysql instance on a dedicated server, you can configure it with your credentials. By default, it uses an sqlite database when running in local                                                                                                                                                                                                                                                             |  |
-| API, Worker | `REDIS_CACHE_HOST`         | `localhost`                        | The hostname of the redis backend used by Celery but also API to cache results                                                                                                                                                                                                                                                                                                                                             |
-| Worker      | `ENABLE_SCREENSHOT`        | `False`                            | If screenshots are enabled, when analyzing the page the image will be generated in the `./screenshot` directory with the image name corresponding to the analysis ID and will be available on the path `/{version}/ecoindexes/{id}/screenshot`                                                                                                                                                                             |
-
-
-## Local development with docker compose
-
-You can use `docker-compose.override.yml` to override the default configuration of the docker-compose.yml file. For example, you can use it to mount your local code in the container and run the server in debug mode.
-
-```bash
-cp docker-compose.override.yml.dist docker-compose.override.yml
-```
-
-Then you can run the server:
-
-```bash
-docker-compose up -d --build
-```
+Here are the environment variables you can configure in your `.env` file:
```

Comment:
It would be helpful to add a brief description of what each environment variable does in the `.env` file. This can help users understand what each variable is used for and how changing it might affect the application. [medium]